Makefile.am

```diff
@@ -60,6 +60,21 @@ GIRS =
 TYPELIBS = $(GIRS:.gir=.typelib)
 endif
 
+# These bits based on gnome:librsvg/Makefile.am
+if ENABLE_RUST
+if RUST_DEBUG
+CARGO_RELEASE_ARGS=
+else
+CARGO_RELEASE_ARGS=--release
+endif
+
+check-local:
+	cd $(srcdir)/rust && CARGO_TARGET_DIR=@abs_top_builddir@/target cargo test
```

Comment:
I think we'd definitely use the Rust unit testing framework, it's really nice.  In this case I explicitly chose not to because the idea is we could reuse the exact same tests that were done for the C implementation.